github/downloader_test.go

```diff
@@ -151,3 +164,107 @@ func TestOnlineOrganizationDownload(t *testing.T) {
 	}
 
 }
+
+func loadReqResp(filepath string, reqResp map[string]string) error {
+	// Open a RO file
+	decodeFile, err := os.Open(filepath)
+	if err != nil {
+		return err
+	}
+	defer decodeFile.Close()
+	// Create a decoder
+	decoder := gob.NewDecoder(decodeFile)
+	// Decode
+	decoder.Decode(&reqResp)
```

Comment:
need to check error. Most probably just `return decoder.Decode(&reqResp)` should be fine